rts/at/bridge/src/main/java/org/jboss/narayana/rest/bridge/inbound/InboundBridgeRecoveryModule.java

```diff
@@ -146,7 +147,7 @@ public void periodicWorkSecondPass() {
                 } while (!finished);
             }
         } catch (Exception e) {
-            LOG.warn(e.getMessage(), e);
+        	RESTATLogger.atI18NLogger.warn_InboundBridgeRecoveryModulegetUidsToRecover(e.getMessage(), e);
```

Comment:
checkstyle reports the use of the tabs as errors (so when I enable the plugin my IDE reports that the file as having errors).

Reply:
+1

Reply:
It is also possible to run the checkstyle plugin directly. Our top level pom includes the maven-checkstyle-plugin. But when I look at that it refers to `<configLocation>wildfly-checkstyle/checkstyle.xml</configLocation>` and we don't have that file in our repo.

The config is also missing the wildfly-checkstyle/suppressions.xml suppressions filter: `https://maven.apache.org/plugins/maven-checkstyle-plugin/examples/suppressions-filter.html`

@ochaloup have you ever run checkstyle from the maven command?

Reply:
@ochaloup I forgot that by default we disable the checkstyle plugin (because much arjuna code dates back many years and we do not want change the orginal code. So the maven command to run the check is:

`mvn checkstyle:checkstyle -Dcheckstyle.skip=false`

and the output will appear in the console.